pkg/kubelet/apis/kubeletconfig/validation/validation.go

```diff
@@ -86,6 +87,13 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	if kc.RegistryPullQPS < 0 {
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: RegistryPullQPS (--registry-qps) %v must not be a negative number", kc.RegistryPullQPS))
 	}
+	// since EvictionHard has a non-empty default, we provide a "none" option that can be used to disable it
+	// when this option is specified, no other thresholds should be present
+	for signal, _ := range kc.EvictionHard {
+		if len(kc.EvictionHard) > 1 && evictionapi.Signal(signal) == evictionapi.SignalNone {
+			allErrors = append(allErrors, fmt.Errorf("invalid configuration: EvictionHard (--eviction-hard) %v may not contain additional thresholds when '%s' is specified", evictionapi.SignalNone))
```

Comment:
I think you forgot an arg to `fmt.Errorf`.  I see `%v` and `%s`, but only `evictionapi.SignalNone`.

Reply:
done